go-controller/pkg/ovn/egressqos_test.go

```diff
@@ -168,6 +170,18 @@ var _ = ginkgo.Describe("OVN EgressQoS Operations", func() {
 				}
 
 				gomega.Eventually(fakeOVN.nbClient).Should(libovsdbtest.HaveDataIgnoringUUIDs(expectedDatabaseState))
+				// Ensure default EgressQoS object is updated with zone success status.
+				gomega.Eventually(func() (string, error) {
+					defaultEq, err := fakeOVN.fakeClient.EgressQoSClient.K8sV1().EgressQoSes(namespaceT.Name).Get(context.TODO(),
+						"default", metav1.GetOptions{})
+					if err != nil {
+						return "", err
+					}
+					if len(defaultEq.Status.Messages) == 1 {
+						return defaultEq.Status.Messages[0], nil
+					}
+					return "", nil
+				}, time.Second).Should(gomega.Equal(types.GetZoneStatus(fakeOVN.controller.zone, egressQoSAppliedCorrectly)))
```

Comment:
looks like a nice candidate to become a function

Reply:
done